x-pack/legacy/plugins/apm/server/lib/service_map/get_trace_sample_ids.ts

```diff
@@ -71,10 +64,9 @@ export async function getTraceSampleIds({
     query.bool.filter.push({ term: { [SERVICE_ENVIRONMENT]: environment } });
   }
 
-  const afterObj =
-    after && after !== 'top'
-      ? { after: JSON.parse(Buffer.from(after, 'base64').toString()) }
-      : {};
+  const afterObj = after
+    ? { after: JSON.parse(Buffer.from(after, 'base64').toString()) }
```

Comment:
What is this doing?

Reply:
it's base64-encoding `after_key` from the composite aggregation. It's not much help in terms of security because it can easily be decoded, but I thought it looked kind of weird just calling JSON.stringify().

Reply:
I get that the after key is an object so we serialize it. But why?

Reply:
This is to help the browser paginate through the composite buckets. If there are more than 1000 connections, pagination kicks in. (we collect 1k connections, and then 10 trace ids per connection per request).

Reply:
Is it to make comparison easier? If so, what about just using `json-stable-stringify` (already part of Kibana deps)?

Reply:
OK with me. But see https://github.com/elastic/kibana/pull/57308#discussion_r377641580 for why I opted for base64. I didn't really have a good reason other than aesthetics 😬